var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -97,6 +98,7 @@ class Sirius(CMakePackage, CudaPackage):
 
     depends_on('spfft@0.9.6: +mpi', when='@6.4.0:')
     depends_on('spfft@0.9.13:', when='@7.0.1:')
+    depends_on('spfft@0.9.13: +single_precision', when='@7.2.5: +single_precision')
```

Comment:
The conflict below is enough so you don't have to repeat the condition
```suggestion
    depends_on('spfft@0.9.13: +single_precision', when='+single_precision')
```

Or even less repetition:

```suggestion
    depends_on('spfft +single_precision', when='+single_precision ^spfft')
```